src/SourceBuild/content/eng/pipelines/templates/stages/vmr-scan.yml

```
@@ -36,6 +36,7 @@ stages:
         ./.dotnet/dotnet darc vmr scan-binary-files
         --vmr "$(Build.SourcesDirectory)"
         --tmp "$(Agent.TempDirectory)"
+        --baseline-file "(Build.SourcesDirectory)/src/installer/src/VirtualMonoRepo/baseline-file.txt"
```

I think could also be simplified thanks to the working directory we set below

```suggestion
        --baseline-file "src/VirtualMonoRepo/baseline-file.txt"
```

---

good point, I also missed the $, don't need it like this